neo/rawio/neuralynxrawio.py

```diff
@@ -133,26 +151,36 @@ def _parse_header(self):
                         'NumADChannels',
                         'DspHighCutFilterType',
                     ]
+
                     d = {k: info[k] for k in keys if k in info}
                     signal_annotations.append(d)
 
                 elif ext in ('nse', 'ntt'):
                     # nse and ntt are pretty similar except for the wavform shape
                     # a file can contain several unit_id (so several unit channel)
-                    assert chan_id not in self.nse_ntt_filenames, \
-                        'Several nse or ntt files have the same unit_id!!!'
-                    self.nse_ntt_filenames[chan_id] = filename
+
+                    # vvv this assert is not informative anymore
+                    # assert chan_id not in self.nse_ntt_filenames, \
+                    #     'Several nse or ntt files have the same unit_id!!!'
+
+                    self.nse_ntt_filenames[uid] = filename
 
                     dtype = get_nse_or_ntt_dtype(info, ext)
-                    data = np.memmap(filename, dtype=dtype, mode='r', offset=HEADER_SIZE)
-                    self._spike_memmap[chan_id] = data
+
+                    if (os.path.getsize(filename)<=HEADER_SIZE):
+                        self._empty_nse_ntt.append(filename)
+                        data = np.zeros((0,), dtype=dtype)
+                    else:
+                        data = np.memmap(filename, dtype=dtype, mode='r', offset=HEADER_SIZE)
+
+                    self._spike_memmap[uid] = data
 
                     unit_ids = np.unique(data['unit_id'])
                     for unit_id in unit_ids:
                         # a spike channel for each (chan_id, unit_id)
-                        self.internal_unit_ids.append((chan_id, unit_id))
+                        self.internal_unit_ids.append((uid, unit_id))
 
-                        unit_name = "ch{}#{}".format(chan_id, unit_id)
+                        unit_name = "ch{}#{}".format(uid, unit_id)
```

Comment:
Let s go back to unit_name = "ch{}#{}".format(chan_id, unit_id) or to
unit_name = "{}ch{}#{}".format(chan_name, chan_id, unit_id)
so the user can make the link with the sofwtare